io.github.suntorytimed.camunda-modeler.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.suntorytimed.camunda-modeler
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: camunda-modeler-wrapper.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+
+add-extensions:
+  com.camunda.Modeler.Plugin:
+    directory: plugins
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+modules:
+  - name: camunda-modeler
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/camunda-modeler
+      - cp -r camunda-modeler/* /app/share/camunda-modeler/
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata
+      - install -Dm755 camunda-modeler-wrapper.sh /app/bin/camunda-modeler-wrapper.sh
+      - install -Dm644 camunda-modeler/support/icon_16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 camunda-modeler/support/icon_48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 camunda-modeler/support/icon_128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - mkdir -p /app/plugins
+    sources:
+      - type: archive
+        only-arches: [x86_64]
+        url: https://github.com/camunda/camunda-modeler/releases/download/v5.32.0/camunda-modeler-5.32.0-linux-x64.tar.gz
```

Comment:
why no building from source?

Reply:
I couldn't successfully build it from source due to issues with npm caching. `npm install --offline` failed for me with an `ENOTCACHED` error, which I couldn't resolve for the npm package. So I couldn't verify a successful build locally. I would prefer building from source though, as it would allow for other supported architectures.

Reply:
@suntorytimed How did you build it from source? Reach out to the project, if you fail to build it.

Reply:
It builds just fine when I build the upstream project outside of flatpak. The issue is the missing cache for a package used in the build.
I did try to 
```
npm install --offline
npm run all
npm run build
```
and it already failed at the `npm install` stept to verify the dependencies. Only in the flatpak build though. Outside of flatpak it works.

Reply:
@bbhtt @hfiguiere are you aware of any such issues regarding building npm/electron applications inside of flatpak with the offline cache? I followed [this documentation](https://docs.flatpak.org/en/latest/electron.html). I could try if I can get it building within a VM and a clean npm, but maybe it is a commonly known problem that already got resolved. Couldn't find anything regarding it though.

Reply:
As far as I can tell it could be an issue with the `flatpak-node-generator`. It doesn't add webpack-sources to the `generated-sources.json`.

Reply:
One issue has been the node_module folder, which existed when I ran `flatpak-node-generator`. Now `generated-sources.json` contains `webpack-sources`. But I still receive the error.